app/views/state_file/questions/income_review/edit.html.erb

```diff
@@ -27,7 +27,11 @@
   <% if current_intake.direct_file_data.fed_unemployment > 0 %>
     <div class="white-group unemployment" id="form1099gs">
       <p class="text--bold spacing-below-0"><%= t(".unemployment_title") %></p>
-      <% if current_intake.state_file1099_gs.count > 0 %>
+      <% if current_state_code == "nj" %>
```

Comment:
[dust] This is another place where you might refactor to remove the state code check

Reply:
thanks! fixed